### ISSUE TITLE
Reader view settings and navigation menus: missing aria-hidden on collapsed sections

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#676]

**Comment by danielweck** [comment]
Related issue: https://github.com/readium/readium-desktop/issues/672
